Vagrantfile

```diff
@@ -0,0 +1,68 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+INVENTORY_FILE = "inventory.vagrant"
+MEMORY_DEFAULT = 384
+HOSTS = [
+  { name: 'jenkins', ip: '172.18.10.9',
+    roles: %w{jenkins-master} },  
+]
+
+def generate_inventory(hosts, file)
+  roles = {}
+
+  File.open(file, "w") do |f|
+    hosts.each do |host|
+      host_string = "#{host[:name]} \
+internal_ip=#{host[:ip]} \
+external_ip=#{host[:ip]}.xip.io \
+ansible_ssh_host=#{host[:ip]}\n"
+
+      f.write(host_string)
+      host[:roles].each do |role|
+        roles[role] ||= []
+        roles[role] << host_string
+      end
+    end
+
+    roles.each do |role, hosts|
+      f.write("\n[#{role}]\n")
+      f.write(hosts.join)
+    end
+  end
+end
+
+generate_inventory(HOSTS, INVENTORY_FILE) if %w{up provision}.include?(ARGV[0])
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.ssh.private_key_path = File.expand_path('~/.vagrant.d/insecure_private_key')
+  config.ssh.insert_key = false
```

Comment:
I think we can omit these two lines and use the defaults. Likewise the `ansible.raw_arguments` below. It was a workaround in alphagov/tsuru-ansible@c2d02f4a1bf5c3a81c0eeee3bc2405ea16e9fc81 for delegation, which we're not using here.